kili/mutations/asset/__init__.py

```diff
@@ -111,7 +111,8 @@ def append_many_to_dataset(
             for i in range(nb_data)
         ]
 
-        import_assets(self.auth, project_id=project_id, assets=assets)
+        result = import_assets(self.auth, project_id=project_id, assets=assets)
```

Comment:
no need to create a new variable `return import_assets(self.auth, project_id=project_id, assets=assets)`